src/main/scala/com/github/swagger/akka/SwaggerHttpService.scala

```diff
@@ -91,7 +91,7 @@ trait SwaggerHttpService extends Directives {
 
   private def filteredSwagger: Swagger = {
     val swagger: Swagger = reader.read(apiClasses.asJava)
-    swagger.setDefinitions(swagger.getDefinitions.asScala.filterKeys(definitionName => !unwantedDefinitions.contains(definitionName)).asJava)
+    swagger.setDefinitions(Option(swagger.getDefinitions).map(_.asScala).getOrElse(Map.empty[String, Model]).filterKeys(definitionName => !unwantedDefinitions.contains(definitionName)).asJava)
```

Comment:
I will once I have found out what is wrong with my own Swagger definition right now. Getting an empty swagger.json and wondering how the tests pass int he first place :D